src/components/lab/NumberInput/NumberInput.scss

```
@@ -0,0 +1,44 @@
+@use '../../variables';
+
+$block: '.#{variables.$ns}number-input';
+
+#{$block} {
+    &_size {
+        &_s {
+            --_--textinput-end-padding: 1px;
+        }
+
+        &_m {
+            --_--textinput-end-padding: 1px;
+        }
+
+        &_l {
+            --_--textinput-end-padding: 3px;
+        }
+
+        &_xl {
+            --_--textinput-end-padding: 3px;
+        }
+    }
+
+    &_view_normal {
+        --_-g-number-input-numeric-arrows-border-color: var(--g-color-line-generic);
```

```suggestion
        --_--arrows-border-color: var(--g-color-line-generic);
```

It's a private var so no need to specify full name